tests/pages/course-admin/submissions-page.js

```diff
@@ -1,5 +1,6 @@
 import { collection, create, clickOnText, triggerable, visitable } from 'ember-cli-page-object';
 import LanguageDropdown from 'codecrafters-frontend/tests/pages/components/course-admin/language-dropdown';
+import StageDropdown from 'codecrafters-frontend/tests/pages/components/course-admin/course-stage-dropdown';
```

Comment:
```suggestion
import CourseStageDropdown from 'codecrafters-frontend/tests/pages/components/course-admin/course-stage-dropdown';
```

Let's keep import names same as the file they're importing from